core/primitives/worker_config.hpp

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Imo config of an implementation is not a primitive and should be placed beside implementation (LocalWorker).